apps/site/assets/css/_events.scss

```diff
@@ -167,21 +164,45 @@ $mobile-control-height: 73px;
   line-height: 22.95px;
   text-transform: uppercase;
   font-weight: 500;
-  padding-top: 21px;
+  padding-top: 4px;
   padding-bottom: 4px;
   @include media-breakpoint-down(xs) {
     font-size: 12px;
-    padding-top: 13px;
+    padding-top: 0;
     padding-bottom: 9px;
   }
 }
 .m-event__title {
   font-size: 18px;
   line-height: 24px;
-  padding-bottom: 25px;
+  padding-bottom: 9px;
   @include media-breakpoint-down(xs) {
     font-size: 14px;
     line-height: 21px;
-    padding-bottom: 19px;
+    padding-bottom: 0;
+  }
+}
+
+.m-event__calendar-add {
+  padding: 10px 14px;
+  margin-left: 30px;
+  font-size: 24px;
+  display: flex;
+  @include media-breakpoint-down(xs) {
+    padding: 12px;
   }
 }
+.m-event__add-text {
+  font-size: 18px;
+  padding-left: 8px;
+  @include media-breakpoint-down(xs) {
+    display: none;
+  }
+}
+.m-event__ended-message {
+  width: 96px;
+  text-align: right;
+  color: $gray;
+  font-size: 18px;
+  font-weight: 500;
+}
```

Comment:
See my comment elsewhere on potentially consolidating some of this -- I think the font-size/weight, width, text-align, left margin, and maybe padding can all be set on the parent div element.